try:
    print(x + 5)
except ValueError:
    print('ValueError')
except TypeError:
    print('TypeError')
